python/django/webappWithTests/Tests/functional_tests/test_functional.py

```diff
@@ -0,0 +1,41 @@
+"""
+This file demonstrates writing functional tests.
+"""
+import pytest
+from selenium import webdriver
+import unittest
+import os
+import sys
+import pytest
+
+class FunctionalTests(unittest.TestCase):
+
+    def setUp(self):
+        options = webdriver.ChromeOptions()
+        options.add_argument('--no-sandbox')
+        self.driver = webdriver.Chrome(os.environ["ChromeWebDriver"], chrome_options=options)
+
+"""
+Sample selenium test. Asserting webapp page title and since webapp deployment takes
+time until the tests are run, we assert default page title until the issue is not 
+with the webapp deployment.
+"""
+    def test_selenium(self):
+        try:
+            webAppUrl = pytest.config.getoption('webAppUrl')
+            response = self.driver.get(webAppUrl)
+            html_source = self.driver.page_source
+            self.assertIn("<title>Home Page - Python Django Application</title>", html_source)
+        except AssertionError:
+            try:
+                self.assertIn("<title>Microsoft Azure App Service - Welcome</title>", html_source)
```

Comment:
add a comment in the code to explain why we are doing this, and also a note to remove it when the deployment issue is fixed

Reply:
Addressed